### PR TITLE
Add bbii-decon recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
Depends on:
- https://github.com/conda-forge/staged-recipes/pull/17727

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@gdellaire, do you agree with being added as a maintainer?

**Comment by gdellaire** [comment]
Sorry

Was not sure I was supposed to respond?

Graham

--
___________________________________________

Graham Dellaire, Ph.D.
Director of Research, Department of Pathology
Professor, Departments of Pathology
and Biochemistry & Molecular Biology
Dalhousie University
P.O. BOX 15000
Halifax, Nova Scotia, Canada, B3H 4R2
Tel: (902)494-4730
Fax: (902)494-2519
E-mail: ***@***.***
Web: dellairelab.medicine.dal.ca<http://dellairelab.medicine.dal.ca>
Twitter: @DocDellaire<https://twitter.com/DocDellaire>
___________________________________________
This e-mail may contain confidential and/or privileged information for the sole use of the intended recipient. Any review or distribution by anyone other than the person for whom it was originally intended is strictly prohibited. If you have received this e-mail in error, please contact the sender and delete all copies. Opinions, conclusions or other information contained in this e-mail may not be that of the organization.

________________________________

File: recipes/bbii-decon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/gdellaire/bbii-decon
+  summary: Projected Barzilai-Borwein Image Deconvolution with Infeasible Iterates (BBii-Decon)
+  license: BSD-3-Clause
```
Comment (jaimergp) [inline recipes/bbii-decon/meta.yaml:39]:
```suggestion
  license: LicenseRef-BBii-Decon-4-Clause
```

**Pull request merged**